data/mods/Aftershock/player/professions.json

```diff
@@ -192,31 +82,6 @@
       "female": [ "boy_shorts", "bra" ]
     }
   },
-  {
-    "type": "profession",
-    "ident": "afs_librarian",
-    "name": "Librarian",
```

Comment:
We could make this an e-ink tablet with a hundred or so sd cards.  Though I'd almost rather mainline this as well.  Most of these professions could be mainlined.